hw7/cmd/task2/main.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
+)
+
+func main() {
+	numbChan := make(chan int)
+	resultChan := make(chan [2]int)
+	doneChan := make(chan bool)
+
+	go numbersGenerator(numbChan, resultChan, doneChan)
+	go findMinMax(numbChan, resultChan)
+
+	<-doneChan
+}
+
+func numbersGenerator(ch chan int, result chan [2]int, done chan bool) {
+
+	for i := 0; i < 7; i++ {
+		num := rand.Intn(100) + 1
+		ch <- num
+	}
+
+	close(ch)
+	fmt.Println(<-result)
+	done <- true
+}
+
+func findMinMax(ch <-chan int, result chan<- [2]int) {
+	min := 999999
```

Comment:
Is there a better way to set the starting point for finding the minimum/maximum element of the chan int?
Something like
 min := mySlice[0] 
in the case of slice of ints

Reply:
you can use `math.MinInt` and `math.MaxInt`